kmod/src/counters.h

```diff
@@ -206,7 +208,9 @@
 	EXPAND_COUNTER(trans_commit_meta_alloc_low)		\
 	EXPAND_COUNTER(trans_commit_sync_fs)			\
 	EXPAND_COUNTER(trans_commit_timer)			\
-	EXPAND_COUNTER(trans_commit_written)
+	EXPAND_COUNTER(trans_commit_written)			\
+	EXPAND_COUNTER(wkic_count_objects)			\
+	EXPAND_COUNTER(wkic_scan_objects)
```

Comment:
Adding to the end of the enumeration of counters needs to update the LAST_COUNTER below.

Reply:
DOH, yes, of course.